src/routes/embedded/auth/import-keyfile/auth-import-keyfile.view.tsx

```diff
@@ -80,55 +77,52 @@ export function AuthImportKeyfileEmbeddedView() {
     };
   }, []);
 
-  return importedTempWalletAddress ? (
-    <Card
-      headerText="Enter Keyfile"
+  return importedWalletAddress ? (
+    <OnboardingCard
+      headerText="Import Keyfile"
       subtitle="Would you like to add this wallet to your account?"
-      footerElement={<WanderFooter />}
-      hasBackButton={true}
-      onBackButtonClick={back}
-      style={{ gap: 24 }}
-      size="auto">
+      onBackButtonClick={() => navigate(`/auth/add-wallet`)}
+      isLoading={ isViewLoading }>
       <Copyable
         isFullWidth
         style={{ padding: 0 }}
         label="Your wallet address"
         onClick={() => {
-          copy(importedTempWalletAddress);
+          copy(importedWalletAddress);
         }}
-        value={importedTempWalletAddress}
+        value={importedWalletAddress}
       />
       <Row>
-        <Button variant="secondary" size="md" onClick={deleteImportedTempWallet}>
+        <Button variant="secondary" size="md" onClick={handleTryAgain} isDisabled={areButtonsDisabled}>
           No, try again
         </Button>
-        <Button variant="primary" size="md" onClick={handleAddWallet} isLoading={loading}>
-          Yes, {authStatus === "noShares" ? "recover" : "add"}
+        <Button variant="primary" size="md" onClick={handleAddWallet} isDisabled={areButtonsDisabled}>
```

Comment:
There should be a loading either on this button or the screen itself  when clicking on `Yes, add` but is missing.

Reply:
Yep, I had `const isViewLoading = isAdding` instead of `const isViewLoading = isUploading`.